aip/0203.md

```diff
@@ -119,10 +119,18 @@ field on a request message (a message that is an argument to an RPC, usually
 ending in `Request`), or a field of a message included within a request
 message.
 
+When a service receives an immutable field in an update request (or similar),
+even if included in the update mask, the service **should** ignore the field if
+the value matches, but **should** error with `INVALID_ARGUMENT` if a change is
+requested.
+
 Potential use cases for immutable fields (this is not an exhaustive list) are:
 
 - Names or IDs which are set on creation and then used as a primary key.
 
+**Note:** Fields which are "conditionally immutable" **must not** be given the
```

Comment:
Is "conditionally immutable" defined elsewhere?  Does that mean things like "GCE instance size", which is immutable while the instance is powered on, but mutable if the instance is powered off?

Reply:
Yes.